data/net/minecraft/world/level/ColorResolver.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/world/level/ColorResolver
```

Comment:
This seems like an unnecessary inclusion although idk if that's enigma's issue or ours?